src/modules/editingTask.js

```diff
@@ -0,0 +1,44 @@
+import modifyTask from './modifyTask.js';
+
+function finishEditing(event) {
+  const selectedElementClass = `.${event.currentTarget.parentNode.parentNode.classList[0]}`;
+  const elementLi = document.querySelector(selectedElementClass);
+  const elementInput = event.currentTarget;
+  elementLi.classList.remove('editing-background');
+  elementInput.classList.remove('editing-background');
+  elementInput.addEventListener('input', modifyTask(elementInput.value, selectedElementClass));
+  const icon = document.querySelector(`${selectedElementClass} > img`);
+  icon.setAttribute('src', './assets/images/dot.png');
+}
+
+function editingTask(event) {
+  document.querySelectorAll('.editing-background').forEach((element) => {
+    element.classList.remove('editing-background');
+  });
+  const elementLi = event.currentTarget;
+  const elementInput = event.currentTarget.firstElementChild.lastElementChild;
+  elementInput.addEventListener('blur', finishEditing);
+  elementLi.classList.add('editing-background');
+  elementInput.classList.add('editing-background');
+  const icon = document.querySelector('.editing-background > img');
```

Comment:
Please consider cleaning the selectors so you can reuse them, seems like there is some repetition.

Reply:
Ok, so I moved the only one that wouldn't affect the functionality. Almost all elements from the DOM are queried using the event. That's why they must be inside the function.